server/rpc/game/game.coffee

```diff
@@ -11749,6 +11763,19 @@ class Fascinated extends Complex
             if pl? && pl.dead
                 @die game, "fascinatesuicide"
 
+# 痛恨の一撃
+class FatalStrike extends Complex
+    cmplType:"FatalStrike"
+    modifyMyVote:(game, vote)->
+        if @sub?
+            vote = @sub.modifyMyVote game, vote
+        vote = @mcall game, @main.modifyMyVote, game, vote
+
+        me = game.getPlayer @id
+        # 自分への投票を稀に100票増やす
+        if  Math.random()<0.05 && !me.isCmplType("VoteGuarded")
+            vote.votes = vote.votes + 100
```

Comment:
This would be very dramatic.
What about to ```addGamelog()``` to his ```Hitokotonushinokami```, then Hitokotonushinokami could get a prize from it.